Source/WebKitLegacy/mac/WebView/WebViewInternal.h

```diff
@@ -126,7 +126,7 @@ WebLayoutMilestones kitLayoutMilestones(OptionSet<WebCore::LayoutMilestone>);
 
 #if PLATFORM(IOS_FAMILY) && ENABLE(DRAG_SUPPORT) && defined(__cplusplus)
 @interface WebUITextIndicatorData (WebUITextIndicatorInternal)
-- (WebUITextIndicatorData *)initWithImage:(CGImageRef)image textIndicatorData:(const WebCore::TextIndicatorData&)indicatorData scale:(CGFloat)scale;
+- (WebUITextIndicatorData *)initWithImage:(CGImageRef)image textIndicatorData:(const WebCore::TextIndicator&)indicator scale:(CGFloat)scale;
```

Comment:
I think this will have to be a RefPtr
```suggestion
- (WebUITextIndicatorData *)initWithImage:(CGImageRef)image textIndicatorData:(RefPtr<WebCore::TextIndicator>&&)indicator scale:(CGFloat)scale;
```

and then you can change your call sites to

```objc
adoptNS([[WebUITextIndicatorData alloc] initWithImage:nil textIndicator:WTFMove(textIndicator) scale:page->deviceScaleFactor()]);
```